test/gtest/uct/uct_test.cc

```diff
@@ -932,15 +940,24 @@ void uct_test::entity::connect_to_sockaddr(unsigned index, entity& other,
     }
 
     /* Connect to the server */
-    ucs_sock_addr_t ucs_remote_addr = remote_addr.to_ucs_sock_addr();
-    uct_ep_params_t params;
-    params.field_mask        = UCT_EP_PARAM_FIELD_IFACE             |
-                               UCT_EP_PARAM_FIELD_USER_DATA         |
+    if (m_cm) {
+        params.field_mask = UCT_EP_PARAM_FIELD_CM                     |
```

Comment:
pls add ucs_print of the value of m_cm
maybe for some reason it's NULL, it can explain the failure